src/main/config/secor.common.properties

```diff
@@ -20,6 +20,8 @@
 # Regular expression matching names of consumed topics.
 secor.kafka.topic_filter=.*
 secor.kafka.topic_blacklist=
+# Pattern example of customtopicsnames: topicName1:customTopicName1;topicName2:customTopicName2;
```

Comment:
Should be comma separated.  I think there is also a map property in secor config which you could follow.

Reply:
Thank you @HenryCaiHaiying . I found the method getPropertyMapForPrefix from SecorConfig class. Its very interesting to use it to build a map of custom name topic configurations. I will use this in my correction.